test_values.go

```diff
@@ -10,6 +10,11 @@ var testCreditCards = map[string]CreditCard{
 	"visa":       CreditCard{Number: "4111111111111111"},
 	"mastercard": CreditCard{Number: "5555555555554444"},
 	"discover":   CreditCard{Number: "6011111111111117"},
+	// See https://developers.braintreepayments.com/reference/general/testing/python#3d-secure
+	"visa_3ds_success":       CreditCard{Number: "4000000000000002"},
+	"mastercard_3ds_success": CreditCard{Number: "5200000000000007"},
+	"visa_3ds_fail":          CreditCard{Number: "4000000000000028"},
+	"mastercard_3ds_fail":    CreditCard{Number: "5200000000000023"},
```

Comment:
These card numbers used to be in the docs, things seems to have changed since I've started working on this. You can find the credit card numbers in the PDF document linked on [this page](https://developers.braintreepayments.com/guides/3d-secure/testing-go-live/php#testing).